project/plugins.sbt

```diff
@@ -1,9 +1,4 @@
 addSbtPlugin("se.marcuslonnberg" % "sbt-docker" % "1.8.0")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.13")
 addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
-addSbtPlugin("org.xerial.sbt" % "sbt-pack" % "0.13")
-addSbtPlugin("io.get-coursier" % "sbt-coursier" % "2.0.3")
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
holy moly

Reply:
will there be a way of generating dependency trees now?

Reply:
Whenever someone discovers this new-to-them toy it's common to install it into this local file. People see "plugin", don't read the docs, and head to the only file they know. [Rawls does it too](https://github.com/broadinstitute/rawls/blob/e0f8b2496c914096dca8028e0cb9708470b9b1e1/project/plugins.sbt#L7).

However, there are two places SBT plugins go: per-project and global. When folks commit global plugins as project plugins it messes up the SBT setup for everyone else who has the plugin installed correctly in global. 😞

From [the docs](https://github.com/sbt/sbt-dependency-graph#usage-instructions):

> sbt-dependency-graph is an informational tool rather than one that changes your build, so you will more than likely wish to install it as a global plugin so that you can use it in any SBT project without the need to explicitly add it to each one.